lib/modules/manager/pixi/index.ts

```diff
@@ -0,0 +1,32 @@
+import type { Category } from '../../../constants';
+import { GitRefsDatasource } from '../../datasource/git-refs';
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { GitlabTagsDatasource } from '../../datasource/gitlab-tags';
+import { PypiDatasource } from '../../datasource/pypi';
+
+export { bumpPackageVersion } from '../pep621/update';
+export { extractPackageFile } from './extract';
+export { updateArtifacts } from './artifacts';
+
+export const supportsLockFileMaintenance = true;
+
+export const url = 'https://github.com/prefix-dev/pixi/';
+export const categories: Category[] = ['python'];
+
+export const defaultConfig = {
+  fileMatch: [
+    '(^|/)pyproject\\.toml$', // `tool.pixi` section
+    '(^|/)pixi\\.toml$', // root object
+  ],
+};
+
+export const supportedDatasources = [
+  PypiDatasource.id,
+  GithubTagsDatasource.id,
+  GithubReleasesDatasource.id,
+  GitlabTagsDatasource.id,
+  GitRefsDatasource.id,
+  GitTagsDatasource.id,
+];
```

Comment:
```suggestion
export const supportedDatasources = [];
```

Reply:
So we fo not need to define this, empty arrat works fine?

Reply:
This suggestion failed tests:

```
   × modules/manager/index > supportedDatasources > has valid supportedDatasources for pixi 10ms
     → expected [] to be an array with at least one item
```

Reply:
This field is for documentation purposes https://docs.renovatebot.com/modules/manager/pep621/#supported-datasources
so it is fine if you add the pypi datasource